services/NotificationService.js

```diff
@@ -17,6 +22,25 @@ module.exports = () => {
                 showUser: params.showUser
             });
 
+            // send push notification if recipient != sender 
+            if (params.notificationFrom !== params.notificationFor) {
+                const user = await US.getUserById(params.notificationFor);
+                const pushToken = user.expoPushToken;
+
+                // make sure recipient has a pushtoken
+                if (Expo.isExpoPushToken(pushToken)) {
+
+                    let message = {
+                        to: pushToken,
+                        body: params.notificationText
+                    }
+                    console.log(message);
+                    let ticket = await expo.sendPushNotificationsAsync([message]);
```

Comment:
Since we won't reference `ticket` after the console.log is removed, we can leave just the await statement and remove it the variable declaration